modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/fragment-content/EditableProcessorContext.js

```diff
@@ -36,18 +36,25 @@ export function EditableProcessorContextProvider({children}) {
 	);
 }
 
+export function useEditableProcessorClickPosition() {
+	const [state] = useContext(EditableProcessorContext);
+
+	return state.editableClickPosition;
+}
+
 export function useEditableProcessorUniqueId() {
 	const [state] = useContext(EditableProcessorContext);
 
-	return state;
+	return state.editableUniqueId;
 }
 
 export function useSetEditableProcessorUniqueId() {
 	const [, dispatch] = useContext(EditableProcessorContext);
 
 	return useCallback(
-		editableUniqueIdOrNull => {
+		(editableUniqueIdOrNull, editableClickPosition) => {
```

Comment:
I think we should set a default value here, something like null in the case we don't receive value, as we are doing for the unique id